controllers/rollingupgrade_controller.go

```diff
@@ -279,7 +289,10 @@ func (r *RollingUpgradeReconciler) WaitForDesiredNodes(ruObj *upgrademgrv1alpha1
 	var err error
 	var ieb *iebackoff.IEBackoff
 	for ieb, err = iebackoff.NewIEBackoff(WaiterMaxDelay, WaiterMinDelay, 0.5, WaiterMaxAttempts); err == nil; err = ieb.Next() {
-		r.populateNodeList(ruObj, r.generatedClient.CoreV1().Nodes())
+		err = r.populateNodeList(ruObj, r.generatedClient.CoreV1().Nodes())
+		if err != nil {
+			return fmt.Errorf("unable to populate node list for: %q, %w", ruObj.Name, err)
```

Comment:
Does it make sense to error out in this case?
Considering this is inside a waiter, it will retry, this way we can avoid transient errors failing the whole thing.
I think may be better to simply log here, and let it timeout if it keeps failing. WDYT?

Reply:
Sure, let's just log it.